bwe/BandwidthEstimator.cpp

```diff
@@ -221,28 +312,37 @@ double BandwidthEstimator::analyseCongestion(double actualDelay, const uint32_t
             _congestion.dip.bandwidthFloorKbps = 0;
         }
     }
+
     return congestionScale;
 }
 
 // in kbps
 double BandwidthEstimator::getEstimate(uint64_t timestamp) const
 {
-    const double estimatedBandwidth = std::min(_state(1), _congestion.dip.bandwidthCapKbps);
+    double estimatedBandwidth = std::min(_state(Bandwidth), _congestion.dip.bandwidthCapKbps);
+    if (_congestion.consecutiveOver < 50)
+    {
```

Comment:
If we have congestion for some time, we do not immediately drop bw as this problem may go away (LTE). We keep the estimate as observed before congestion for 50pkt. (Usually about 0.5s). If it does not improve by then we let it drop to the model estimate.